### PR TITLE
Itemremove for CTW

### PR DESCRIPTION
**Pull request by Owehttamy**

### CODE REVIEW

File: CTW/A_New_Day/map.json [hunk 1]
```diff
@@ -77,8 +77,7 @@
 		}
 	],
 	"itemremove": [
-		"iron sword", "bow", "iron pickaxe", "iron axe", "leather helmet", "oak planks", "cooked beef", 
-		"arrow", "golden apple", "chainmail chestplate", "chainmail leggings", "leather boots", "beacon"
+		"iron sword", "bow", "iron pickaxe", "iron axe", "oak planks", "cooked beef", "golden apple", "chainmail chestplate", "chainmail leggings", "leather boots", "beacon"
```
Comment (BennyDoesTheStuff) [inline CTW/A_New_Day/map.json]: What's the reasoning for removing "arrow" and "golden apple" from the item remove? Everyone in the game is already getting 32 arrows in their inventory and getting 32 every time you kill someone probably is a bit too much. Additionally, you already get a golden apple on kill so I don't see the need to also get any golden apples the player had (every player starts with 3 golden apples, that also seems excessive).

Comment (Owehttamy) [reply CTW/A_New_Day/map.json]: Got it, so if you already get a bunch of the item you don't need it. I'll do that from now on.

**Comment by Owehttamy** [comment]
done

**Comment by BennyDoesTheStuff** [comment]
Way too many commits + You already edited the item remove for Golden Drought so I'm not sure why you're trying to edit it again lol

**Comment by Owehttamy** [comment]
But why tho
I fixed the map's item removal
Also you guys told me to do the commits in one pull request because doing it per every map was too spammy

**Comment by Owehttamy** [comment]
I did an oopsie

**Comment by CrazyisCreeps** [comment]
Yea uh there's no more "merge" button

**Comment by Owehttamy** [comment]
Welp I might just have to redo them lmao

Won't be that bad since it's only a few maps and I already have the code here

**Pull request closed without merging**